stablehlo/dialect/VhloBytecode.cpp

```diff
@@ -220,6 +214,12 @@ enum AttributeCode {
   ///   }
   kStringAttr = 22,
 
+  ///   TensorAttr {
+  ///     type: Type
+  ///     data: blob
+  ///   }
+  kTensorAttr = 17,
```

Comment:
Merge the other bytecode modifications then lets renumber since we can break still. Could open an issue to follow up.

Reply:
Yes, this is done in the final PR of this chain ("Audit StablehloAttrs.td").